### ISSUE TITLE
Printing probably doesn't work

### ISSUE THREAD

**Issue by guihkx** [flathub/br.gov.fazenda.receita.irpf#33]
I don't have a printer to test this at the moment, but I'm fairly certain that printing doesn't really work, mainly because the `socket=cups` permission was never given in the first place... 😅 

Steps to reproduce:

1. Have a *declaração* at least partially filled
2. Hit <kbd>Ctrl</kbd>+<kbd>P</kbd> to open the printing window
3. Make sure the `Visualizar` radio button is the one selected
4. Hit `Ok`
5. Click the small printer button, located at the top left corner
6. Error pops up:
  ![image](https://user-images.githubusercontent.com/626206/194819668-7fbb6ce0-6803-4056-82b7-0404cbc404e8.png)

---

Possible fixes:

* Add the `--socket=cups` permission ([not a guaranteed fix](https://github.com/flatpak/flatpak/issues/3355)):

  ```
  flatpak override --user --socket=cups br.gov.fazenda.receita.irpf
  ```

I'll try to get a printer to test the fix, but it'd be great to get a confirmation by someone who already owns a printer.

**Comment by guihkx** [comment]
@astroty Have you tried adding the CUPS permission? I.e.:

```
flatpak override --user --socket=cups br.gov.fazenda.receita.irpf
```

Then restart the app.

**Comment by astroty** [comment]
I've tried it through flatseal and gave it pretty much every permission and it didn't seem to wanna work
"Houve uma falha na comunicação com a impressora e não foi possível concluir a impressão.
Recomendamos salvar a declaração em PDF e imprimi-la novamente."
Or, if instead of clicking "Visualisar", I choose "Imprimir" instead, it gives me this different error message:
![image](https://user-images.githubusercontent.com/1356321/233404346-140ca50a-6bda-4a34-a4e6-edd79c5f1549.png)

**Comment by guihkx** [comment]
Yeah, that's unfortunate. The error messages you're getting seem to match the ones from the issue I linked in my first post. I'm not aware of a workaround yet.
